docs/src/docs/cli/installation.md

```diff
@@ -2,101 +2,144 @@
 import vGaTrack from '@theme/directives/ga'
 </script>
 
-# Installation Guide for `kit` CLI
+# Installing Kit
 
-## Installation from GitHub Releases
-Welcome to the installation guide for the `kit`! This guide is designed to help you quickly and easily install the `kit` on your machine.
+This page includes instructions for:
 
-### Step 1: Downloading the `kit`
+* Installing from [MacOS packages](#🍎-macos-package-download)
+* Installing from [Windows packages](#🪟-windows-package-download)
+* Installing from [Linux packages](#🐧-linux-package-download)
+* Building from [source code](#🛠️-install-from-source)
 
-To begin, you will need to download the latest version of `kit`. You can find the most recent release on the official GitHub releases page:
+## 🍎 MacOS Package Download
```

Comment:
This is not just download it also has instruction to install. I think the heading should reflect that.